naresh/src/main/java/com/lftechnology/phpjava/menu/MenuReader.java

```diff
@@ -0,0 +1,12 @@
+package com.lftechnology.phpjava.menu;
+
+/**
+ * Created by naresh on 5/16/16.
+ */
+public interface MenuReader {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Annotate the "MenuReader" interface with the @FunctionalInterface annotation [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.java.lftechnology.com:9000/coding_rules#rule_key=squid%3AS1609)